sites/partners/src/lib/helpers.ts

```diff
@@ -147,7 +147,13 @@ export const createTime = (
 export const createDate = (formDate: { year: string; month: string; day: string }) => {
   if (!formDate || !formDate?.year || !formDate?.month || !formDate?.day) return null
 
-  return dayjs(`${formDate.year}-${formDate.month}-${formDate.day}`, "YYYY-MM-DD").toDate()
+  const createdDate = new Date(
```

Comment:
For some reason (TBD), month is 0 indexed and day/year are the actual value ([docs](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date/Date#monthindex))